codelabs/gRPC_Cpp_OpenTelemetry_Plugin/README.md

```diff
@@ -0,0 +1,18 @@
+# gRPC C++ OpenTelemetry Plugin
+
+Get hands-on with gRPC's OpenTelemetry plugin for C++ in this interactive codelab! <!-- TODO(arvindbr8): Insert link once codelab is published. -->
+
+
+Designed for developers already familiar with gRPC and wanting to learn how to instrument their gRPC usage with OpenTelemetry.
+
+#### You'll learn how to: 
+- setup gRPC's OpenTelemetry plugin in C++.
```

Comment:
```suggestion
- setup gRPC's OpenTelemetry plugin in gRPC C++.
```

Reply:
why repeat gRPC?

Reply:
Because we are setting up gRPC Otel plugin in gRPC C++ and not C++?  Every comment here is really optional